sllurp/inventory.py

```diff
@@ -132,7 +132,9 @@ def main():
             logger.info('selected recommended Tari of %d for %s', args.tari,
                         args.modulation)
 
+    # build the antennas tuple and transmit power map {antenna_id: tx power}
     enabled_antennas = map(lambda x: int(x.strip()), args.antennas.split(','))
+    tx_power_map = dict([(antid, args.tx_power) for antid in enabled_antennas])
```

Comment:
Python 2 intertia :)
Dict comprenhension is available in 2.7 too, so it will ok AFAIK